src/gpu/gpu_impl_list.hpp

```diff
@@ -74,7 +74,7 @@ namespace gpu {
 #if defined(DNNL_WITH_SYCL) \
         && ((DNNL_GPU_VENDOR == DNNL_VENDOR_GENERIC) \
                 || (DNNL_GPU_VENDOR == DNNL_VENDOR_NVIDIA) \
-                || (DNNL_AMD_ENABLE_SYCL_KERNELS == 1))
+                || defined(DNNL_AMD_ENABLE_SYCL_KERNELS))
```

Comment:
If we don't care about the value then let's remove it from the definition as well:
https://github.com/uxlfoundation/oneDNN/blob/48e7e7379e0b07fd541012ceed837f6bbde21ebc/cmake/options.cmake#L333

Reply:
Makes sense to me, removed, thanks.